### ISSUE TITLE
CKAN 2.9: Review template overrides

### ISSUE THREAD

**Issue by chris48s** [okfn/ckanext-unhcr#536]
There are many upstream templates which we are overriding. In some cases there may be upstream template changes that mean we need to update our overrides

**Comment by chris48s** [comment]
Most of this is now done in #552 #553 and #554

The last thing I need to deal with is backporting the changes from https://github.com/ckan/ckan/commit/de7ba5dde39f1df46c95d0fa87f0cfb5163550bd onto our custom version of `templates/organization/member_new.html` but this is slightly more involved so I will do it as another separate PR.

Then I think once that is done I will handle the rest of the template updates under https://github.com/okfn/ckanext-unhcr/issues/542 as that is the remaining category of thing that needs doing.

**Comment by chris48s** [comment]
#556 wraps this one up

**Comment by chris48s** [comment]
remaining template work is will be handled under #530
